### PR TITLE
chore: document log threshold configuration

### PR DESCRIPTION
**Pull request by jamesonwilliams**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -461,6 +461,17 @@ output to a file, and grep through it.
...
 ```
 
+### Getting More Output
+
+The Amplify Android library emits logs while it is running on a device
+or emulator. By default, debugging and verbose logs are not output.
+However, you can change the log treshold at runtime, by explicitly
+configuring a logging plugin:
+```kotlin
+Amplifly.addPlugin(AndroidLoggingPlugin(LogLevel.VERBOSE))
```
Comment (richardmcclellan) [inline CONTRIBUTING.md:471]:
```suggestion
Amplify.addPlugin(AndroidLoggingPlugin(LogLevel.VERBOSE))
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -461,6 +461,17 @@ output to a file, and grep through it.
 ./gradlew --debug 2>&1 > debugging-the-build.log
 ```
 
+### Getting More Output
+
+The Amplify Android library emits logs while it is running on a device
+or emulator. By default, debugging and verbose logs are not output.
+However, you can change the log treshold at runtime, by explicitly
```
Comment (richardmcclellan) [inline CONTRIBUTING.md:468]:
```suggestion
However, you can change the log threshold at runtime, by explicitly
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -461,6 +461,17 @@ output to a file, and grep through it.
 ./gradlew --debug 2>&1 > debugging-the-build.log
 ```
 
+### Getting More Output
+
+The Amplify Android library emits logs while it is running on a device
+or emulator. By default, debugging and verbose logs are not output.
```
Comment (richardmcclellan) [inline CONTRIBUTING.md:467]:
```suggestion
or emulator. By default, debug and verbose logs are not output.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -461,6 +461,17 @@ output to a file, and grep through it.
...
+However, you can change the log treshold at runtime, by explicitly
+configuring a logging plugin:
+```kotlin
+Amplifly.addPlugin(AndroidLoggingPlugin(LogLevel.VERBOSE))
+// ... Add more plugins only *after* setting the log plugin.
```
Comment (richardmcclellan) [inline CONTRIBUTING.md:472]: Hmm, what happens if I add the logging plugin last?  I didn't think order mattered.

Comment (jamesonwilliams) [reply CONTRIBUTING.md:472]: The other plugins may use logging while they are initializing themselves. The lifecycle of logging doesn't fit nicely into the "configure all at once" model.

**Comment by jamesonwilliams** [comment]
Thanks @richardmcclellan for catching all of the typos. :-)

**Review by richardmcclellan — APPROVED** [review]

**Pull request merged**